sndev

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```

Comment:
Some shell options I like to set:

`set -e` to exit immediately on error which is more aligned with how scripts are expected to execute

> Exit immediately if a pipeline (see [Pipelines](https://www.gnu.org/software/bash/manual/html_node/Pipelines.html)), which may consist of a single simple command (see [Simple Commands](https://www.gnu.org/software/bash/manual/html_node/Simple-Commands.html)), a list (see [Lists of Commands](https://www.gnu.org/software/bash/manual/html_node/Lists.html)), or a compound command (see [Compound Commands](https://www.gnu.org/software/bash/manual/html_node/Compound-Commands.html)) returns a non-zero status.

`set -x`: print command output (probably not useful here, simply using `echo` to tell what we're doing is fine but just fyi)

> Print a trace of simple commands, for commands, case commands, select commands, and arithmetic for commands and their arguments or associated word lists after they are expanded and before they are executed.

`set -r`: enter restricted mode

> Enable restricted shell mode. This option cannot be unset once it has been set.

I actually never used this but just found out about it and found it interesting enough to mention it here. Probably also not needed. We're not running `rm` anywhere so [this](https://github.com/valvesoftware/steam-for-linux/issues/3671) shouldn't be possible.

There is also a name for every option that one can use with `set -o`.

```suggestion

set -o errexit  # exit immediately if a command fails

```

source: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html